src/Microsoft.AspNet.OData/ODataOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="ODataOptions.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using Microsoft.AspNet.OData.Query;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Provides programmatic configuration for the OData service.
+    /// </summary>
+    public class ODataOptions
```

Comment:
I can do that if it makes sense - I only used `ODataOptions` because I saw it defined in the AspNetCore implementation and so thought it was the correct pattern. I'll see what it looks like to switch to this existing thing.